api/web/templates/web/project_view.html

```diff
@@ -81,27 +81,28 @@ <h3 class="mb-0">
                               </h3>
                             </div>
                             <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion">
-                              <div class="card-body" id="chartId" style="width:100%; height:400px;"></div>
+                              <div class="card-body" id="chartId" style="width:100%; height:400px; text-transform: capitalize;"></div>
                             </div>
                           </div>
 
                           <div class="card">
                             <div class="card-header bg-gradient-light" id="headingTwo">
                               <h3 class="mb-0">
-                                <button class="btn btn-link collapsed text-dark" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+                                <button class="btn btn-link collapsed text-dark" onclick="hot.render()" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
                                   Table
                                 </button>
                               </h3>
                             </div>
-                            <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
-                              <div class="card-body" id="data-table" style="padding-top: 0px;padding-left: 0px;padding-right: 0px;padding-bottom: 0px; width: 100%;"></div>
+                            <div id="collapseTwo" class="collapse clearfix" aria-labelledby="headingTwo" data-parent="#accordion">
+                                <button id="export-file" class="intext-btn btn btn-outline-primary float-xl-right" style="margin: 10px; margin-left: 0px;"><span class="fas fa-download" style="margin-right: 10px;"></span>Download CSV</button>
```

Comment:
Adicionei um spinner para mostrar ao usuário que a tabela está sendo renderizada.

```suggestion
                                <button id="export-file" class="intext-btn btn btn-outline-primary float-xl-right" style="margin: 10px; margin-left: 0px;"><span class="fas fa-download" style="margin-right: 10px;"></span>Download CSV</button>
                                <div class="d-flex justify-content-center">
                                    <div id="data-table-loading" style="position: absolute; margin-top: 10px;" class="spinner-border text-primary" role="status">
                                        <span class="sr-only">Loading...</span>
                                    </div>
                                </div>
```